### PR TITLE
Refactor/#1432-make this final field static too

### PR DESCRIPTION
**Pull request by jianglinyao1996**
**What's in the PR**
* make this final field static too

**How to test manually**
* ...

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

**Comment by jianglinyao1996** [comment]
The final field is unique in one object class. The static final field is unique in multiple objects. If it used as a final constant, you have to create an object when it is referenced by other classes, which will take up unnecessary space. If you add static, it will occupy a space when compiling, and it will not take up space at other times.

File: inception-search-mtas/src/test/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndexTest.java [hunk 1]
```diff
@@ -468,7 +468,7 @@ public void testAnnotationQuery() throws Exception
         @Autowired
         ApplicationEventPublisher applicationEventPublisher;
 
-        private final String temporaryFolderPath = "target/MtasDocumentIndexTest";
+        private static final String temporaryFolderPath = "target/MtasDocumentIndexTest";
```
Comment (reckart) [inline inception-search-mtas/src/test/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndexTest.java]: final static constants should be written as `CAPITAL_LETTERS`

**Comment by munterkalmsteiner** [comment]
@jianglinyao1996 please address @reckart comment above. Then I'll suggest to approve the pull request.

**Comment by munterkalmsteiner** [comment]
@jianglinyao1996 do you intend to address reckart's comment?

**Pull request closed without merging**